_docs/2.0.0/plugins.md

```diff
@@ -1,11 +1,11 @@
 ---
 layout: docs
-title: Built-in Plugins
+title: Plugins
 version: '2.0.0'
 categories:
     - '2.0.0'
+redirect_from: /docs/plugins-built-in
```

Comment:
```suggestion
redirect_from: /docs/plugins
```
Btw. where are these paths coming from anyway?

Reply:
These paths are set in `_config.yml` in the root of `gh-pages` branch. They are defined by the collection `docs`. 
The default/latest path/permalink is something @benedeki asked for so we could put non-changeable links to the latest docs.

Does this make sense?